modules/client-configuration/pages/autoinst-profiles.adoc

```diff
@@ -61,31 +70,162 @@ This can overwrite your customization.
 Verify new or changed settings before saving them, by navigating to the [guimenu]``Partitioning`` tab.
 ====
 
-You can change the details and settings of your autoinstallation profiles by navigating to menu:Systems[Autoinstallation > Profiles] and clicking the name of the profile you want to edit.
+The kernel options from the distribution and the profile are combined.
+
+You can change the details and settings of your autoinstallation profiles by navigating to menu:Systems[Autoinstallation > Profiles] and clicking the name of the profile you want to edit. Alternatively, you may navigate to menu:Systems[System List], select the client you want to provision, and navigate to the menu:Provisioning[Autoinstallation] subtab.
+
+
+[[autoyast]]
+== AutoYast Profiles
+
+An {ay} profile consists in a [guimenu]``Label`` that identifies the profile, an [guimenu]``Autoinstall Tree`` that points to an autoinstallation distribution, various options, and, most importantly, an {ay} installation file.
+
+This "control" file is an XML file that give directions to the {ay} installer. You will find the full syntax of {ay} installation files at https://doc.opensuse.org/projects/autoyast/#cha-configuration-installation-options.
```

Comment:
```suggestion
This file is stored as an XML file that gives directions to the {ay} installer. 
For more information about the syntax used in {ay} installation files, see https://doc.opensuse.org/projects/autoyast/#cha-configuration-installation-options.
```

Reply:
"control file" was intentionally mentioned because it's how it is called in the AutoYast documentation.